### DISCUSSION TITLE
Cannot convert a Symbol value to a string

### DISCUSSION THREAD

**Discussion by kilroyFR** [serenity-is/Serenity#6997] [General]
hey all, i am currently moving a former 3.x app to 6.x.

I am getting an issue at login.
![image](https://github.com/serenity-is/Serenity/assets/31513956/d579c7d5-37ec-47d0-a027-41c2756f1820)

I dont understand th issue, it hangs on accountPanelTitle.tsx not finding Texts.Forms.SiteTitle

![image](https://github.com/serenity-is/Serenity/assets/31513956/70eebcce-b6a3-468b-88ed-3eb70ddf559c)

while it actually exists

![image](https://github.com/serenity-is/Serenity/assets/31513956/19426afa-7ae4-415e-bd88-f2ee1af8eab1)

Any help is appreciated

**Comment by volkanceylan** [comment]
You should check your Serenity.Scripts version, probably your sergen and Serenity.Scripts versions are mismatching

**Reply by kilroyFR** [reply]
Thanks Volkan, i did not change any version since StartSharp generation.

Here is what i got : exception raising when trying to load "Forms.SiteTitle.length".
Symptom is that user login panel is not displayed (blank screen after exception popup).
I am pretty sure it's minor thing to fix but really stuck on this.
Thanks for any more help, 

![image](https://github.com/serenity-is/Serenity/assets/31513956/6f7806a5-1fb8-4d25-8adf-a85ab8843829)

**Comment by volkanceylan** [comment]
If you did not change anything, and using same version of both sergen, Serenity.Scripts, jsxdom etc. it should work, as we are creating demo from same version. Check your sergen.json to see which version your app is created from and check sergen and Serenity.Scripts versions.  Also you may change that line to {localText("Forms.SiteTitle")} and try that way.

**Comment by volkanceylan** [comment]
You told that you are moving an app from 3.x to 6.x so you should have updated some parts. Anyway, remove the localTextPackages setting from appsettings.json if you have it there.
